airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/exceptions.py

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+class ExceptionWithDisplayMessage(Exception):
```

Comment:
Where is this used? I don't see any usage

Reply:
This is used in the [facade's `get_error_display_message` so exceptions with a display message can be rendered to the user](https://github.com/airbytehq/airbyte/pull/30113/files#diff-93127bface0b323fe43b21cdb8fb14493dd465995b085a4f81647f3697930bddR191-L123). 

Currently, the only way such exceptions can be raised is through the [facade's read method](https://github.com/airbytehq/airbyte/pull/30113/files#diff-93127bface0b323fe43b21cdb8fb14493dd465995b085a4f81647f3697930bddR236-R240) and [check_availability](https://github.com/airbytehq/airbyte/pull/30113/files#diff-93127bface0b323fe43b21cdb8fb14493dd465995b085a4f81647f3697930bddR326-R329). 

More use cases will come up as we start implementing the partition objects. The goal is for them to raise an exception with the message to display